Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -952,7 +968,7 @@ internal enum EventTriggering {
                     return this.Caption;
 
                 case __VSHPROPID.VSHPROPID_ProjectDir:
-                    return this.ProjectFolder;
+                    return this.ProjectHome;
```

Comment:
Are you sure about this? ProjectDir is supposed to be the directory of the project file. What other components assume that all the source files are beneath this directory?